### DISCUSSION TITLE
Implement solving nonlinear or stochastic problem (cost optimization)

### DISCUSSION THREAD

**Discussion by universePr** [augustoproiete/microsoft-solver-foundation-express-nuget#37] [Q&A]
I use Solver Foundation to solve linear problems and it works perfectly.
But for methods such as non-linear or stochastic methods, there is no documentation or sample code.
I want to use grg non-linear method in Excel in the code.
can you help me?
or help for use Stochastic Directive

**Comment by augustoproiete** [comment]
Hi @universePr there's very little documentation available these days... Please see this answer for some possible pointers: https://github.com/augustoproiete/microsoft-solver-foundation-express-nuget/discussions/9#discussioncomment-940042
